guides/v2.2/cloud/trouble/trouble_ce-creds.md

```diff
@@ -97,7 +97,7 @@ To resolve the issue with credentials:
 
    {% collapsible Click to show/hide snippet %}
```

Comment:
Remove this line and the `endcollapsible` below...

Reply:
thanks, all removed. i'd look into why this is happening with the collapsible tag but i don't think it's really even needed here anyways. should be all set now. thanks for the `terminal` recommendation. how are you viewing/previewing the markdown files locally to confirm they look ok? i'm using macdown but apparently it's not a 1:1 preview with how github displays markdown.

Reply:
I am building the site locally using a custom `_config.local.yaml` to exclude all guides except for Cloud. Makes it faster. 😉

Reply:
https://github.com/magento/devdocs#building-this-site